### ISSUE TITLE
If there's a warning about missing keys above, please disregard

### ISSUE THREAD

**Issue by ifromeast** [SAI990323/TALLRec#5]
When I train samples=2048, the ERROR occurs that
```
{'train_runtime': 11868.7058, 'train_samples_per_second': 69.022, 'train_steps_per_second': 0.27, 'train_loss': 0.2145603246986866, '
epoch': 240.0}                                                                                                                       
 60%|████████████████████████████████████████████████████▊                                   | 1920/3200 [3:17:48<2:11:52,  6.18s/it]
                                                                                                                                     
 If there's a warning about missing keys above, please disregard :)                                                                  
                                                                                                                                     
 If there's a warning about missing keys above, please disregard :)
```
Is there something wrong with the `movie` dataset, and how can I solve it?

**Comment by SAI990323** [comment]
" If there's a warning about missing keys above, please disregard :)   " is not an error. You finished the training phrases.

**Comment by ifromeast** [comment]
> " If there's a warning about missing keys above, please disregard :) " is not an error. You finished the training phrases.

Why the progress bar shows just 60%?

**Comment by SAI990323** [comment]
> > " If there's a warning about missing keys above, please disregard :) " is not an error. You finished the training phrases.
> 
> Why the progress bar shows just 60%?
As shown in the [code](https://github.com/SAI990323/TALLRec/blob/28dca7b9555fe7a3f66fbfd9401dc64df2598e1a/finetune_rec.py#LL278C1-L278C72),  it meets the condition of the early stop.

**Comment by ifromeast** [comment]
> > > " If there's a warning about missing keys above, please disregard :) " is not an error. You finished the training phrases.
> > 
> > 
> > Why the progress bar shows just 60%?
> > As shown in the [code](https://github.com/SAI990323/TALLRec/blob/28dca7b9555fe7a3f66fbfd9401dc64df2598e1a/finetune_rec.py#LL278C1-L278C72),  it meets the condition of the early stop.

![image](https://github.com/SAI990323/TALLRec/assets/31949186/95ab6efb-2797-4cb0-8145-15c454d67cbf)
and this eval/loss means overfit ?

**Comment by SAI990323** [comment]
> > > > " If there's a warning about missing keys above, please disregard :) " is not an error. You finished the training phrases.
> > > 
> > > 
> > > Why the progress bar shows just 60%?
> > > As shown in the [code](https://github.com/SAI990323/TALLRec/blob/28dca7b9555fe7a3f66fbfd9401dc64df2598e1a/finetune_rec.py#LL278C1-L278C72),  it meets the condition of the early stop.
> 
> ![image](https://user-images.githubusercontent.com/31949186/242841114-95ab6efb-2797-4cb0-8145-15c454d67cbf.png) @SAI990323 and this eval/loss means overfit ?

I think it should be overfitting.
